content/docs/user_guide/FAQ.md

```diff
@@ -35,6 +35,11 @@ If you’re having problems running a container with `runsc` it’s most likely
 to a compatibility issue or a missing feature in gVisor. See
 [Debugging](../debugging/).
 
+### When I run my container, docker fails with `open /run/containerd/.../<containerid>/log.json: no such file or directory`.
+
```

Comment:
Can you add a TODO comment to remove this once we fix the bug?

Reply:
Markdown is tricky with comments since they aren't portable across parsers. I'll try to figure out the best one to use for hugo.

Reply:
Added a comment with link to issue